.changeset/chatty-flowers-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
```

Comment:
Since this is something new and not a bug fix, should it be `minor`/`major`?

Reply:
doesn't matter since it will go in 3.0 anyway, but i think it can be seen as `patch` since it "fixes RSC support"?

it's certainly not a new feature that we implemented ourselves.